### ISSUE TITLE
dyld: Library not loaded: @rpath/libswiftCore.dylib

### ISSUE THREAD

**Issue by leogdion** [iainsmith/swift-docker#4]
> dyld: Library not loaded: @rpath/libswiftCore.dylib
>   Referenced from: /usr/local/bin/swift-docker
>   Reason: image not found

**Comment by mxcl** [comment]
This happens on 10.14.4 and above, the formula builds with `-static-stdlib` which will produce an invalid binary on these platforms (can't recall why†).

The solution is to not use this flag there @iainsmith see my formula for `swift-sh` where I do a similar work around:

https://github.com/mxcl/homebrew-made/blob/master/swift-sh.rb

> † well… it's because Swift is ABI safe on 10.14.4 and above, but why exactly this means `static-stdlib` *fails*, I don’t know.

**Comment by mxcl** [comment]
This tool is not maintained, unfortunately. I have a script I use instead so my motivation to fix it up is low. I wanted to use this instead of the script.

Main big problem right now is it targets images from a docker repo that is not up-to-date, so you can’t go above Swift 4.0.

**Comment by leogdion** [comment]
That's what I figured. Thanks for the update.

I've been running this to test on linux from macOS:
```
docker run --rm \
    --volume "$(pwd):/package" \
    --workdir "/package" \
swift \
 /bin/bash -c \
    "swift package resolve && swift test --build-path ./.build/linux"
```

**Comment by iainsmith** [comment]
Hi @leogdion / @mxcl. Just finished up some other projects, so hoping to get this upto date over the next couple of weeks.

Thanks for reporting the issue.

**Comment by iainsmith** [comment]
Hey @leogdion. This should all now be resolved with the 0.2.0 release. Closing for now, but please reopen if you hit any issues.
